### DISCUSSION TITLE
Caution: Lolin C3 Pico 1.0.0 board is not Qwiic compatible

### DISCUSSION THREAD

**Discussion by mkellner** [Moddable-OpenSource/moddable#1157] [Announcements]
I bought this board and it has a connector that fits the Qwiic or STEMMA style connector. If you plug a your sensor, you will likely fry it like I did.

Please take care.

![image](https://github.com/Moddable-OpenSource/moddable/assets/6822704/2bfd0af7-dc2a-4d72-be17-56319d6d751e)
![image](https://github.com/Moddable-OpenSource/moddable/assets/6822704/b25a4d77-5c59-4d38-8537-5270c121b219)
![image](https://github.com/Moddable-OpenSource/moddable/assets/6822704/4c075f76-03b3-497b-8a85-ca0e78dee1d2)

**Comment by tve** [comment]
Thanks for the heads-up! For reference, QWIIC uses GND / 3.3V / SDA / SCL.
